core/templates/domain/question/editable-question-backend-api.service.ts

```diff
@@ -87,9 +87,8 @@ export class EditableQuestionBackendApiService {
       body.append('payload', JSON.stringify(postData));
       let filenames = imagesData.map(obj => obj.filename);
       let imageBlobs = imagesData.map(obj => obj.imageBlob);
-      for (let idx in imageBlobs) {
-        body.append(filenames[idx], imageBlobs[idx]);
-      }
+      body.append('filename', filenames[0]);
+      body.append('image', imageBlobs[0]);
```

Comment:
Can there be more than one images here?

Reply:
@aks681 An interaction can have one image only for a question, so I kept it as the first index.

Reply:
Yes, but you can add images to the question content, wherever a rich text editor is present, so are they uploaded here as well?

Reply:
Ok yes, that is possible. Then I will the send the entire array of images, in the next commit